webview/src/plots/components/comparisonTable/styles.module.scss

```diff
@@ -170,12 +151,45 @@ $gap: 4px;
   max-height: 0;
 }
 
-.cell img {
+.image {
   width: 100%;
   height: auto;
   vertical-align: middle;
 }
 
+.imageWrapper {
+  width: 100%;
+  display: block;
+  padding: 0;
+  border: 0;
+}
+
+.noImageContent {
+  padding-top: 25%;
+  padding-bottom: 25%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.multiImageWrapper {
+  .image,
+  .noImageContent {
+    height: 380px;
```

Comment:
Had to have a fixed height for multi images since we need all images in the multi plots to be the same height:

https://github.com/iterative/vscode-dvc/assets/43496356/96067397-e567-4f23-be5a-369d156b39a4

There should be some ways to at least get an average height based off the images instead of just using a fixed number but I figured it could be saved for a followup.

Reply:
> This looks really weird. Definitely need to fix.

Understood! I didn't think it looked that bad, but if you think it looks really weird, I'm happy to work on fixing it before merging.

Reply:
Can still be done in follow-up. Just make sure we don't move on before it is fixed 🙏🏻.